common/src/main/java/org/iris_events/annotations/Message.java

```diff
@@ -69,4 +69,15 @@
      * they reach the queue.
      */
     boolean persistent() default false;
+
+    /**
+     * Defines the type of RPC response for this message.
+     * For messages with this property defined {@link org.iris_events.producer.EventProducer#sendRpcRequest(Object, Class)} can
+     * be used.
+     * The first parameter of the sendRpcRequest should be of type annotated with this instance of {@link Message} annotation,
```

Comment:
This is a bit confusing, is it possible to simplify this javadoc?

Reply:
I will do my best to simplify.